HelpfulItem.h

```diff
@@ -9,7 +9,6 @@
 
```

Comment:
Don't include what you don't use
If you aren't using anything from these header files, don't include them
- [ ] revise